src/cmd/services/m3query/config/config.go

```diff
@@ -184,16 +214,6 @@ func (c Configuration) LookbackDurationOrDefault() (time.Duration, error) {
 	return v, nil
 }
 
-// LimitsOrDefault returns the specified limit configuration if provided, or the
-// default value otherwise.
-func (c Configuration) LimitsOrDefault() *LimitsConfiguration {
```

Comment:
@richardartoul I reverted your changes around this. The previous default was to have limits disabled, which I think is still probably the better default (at least for the moment). The `defaultLimitsConfiguration` variable was an unfortunate holdover--I didn't catch it in the original PR, and forgot to remove it even though it was unused. I'm assuming your intent with introducing `LimitsOrDefault()` was to fix the unused variable--let me know if that wasn't the case!